lockfile_generator/Cargo.toml

```diff
@@ -13,3 +13,4 @@ anyhow = "1.0.75"
 glob = "0.3.1"
 thiserror = "1.0.49"
 tempfile = "3.3.0"
+dunce = "1.0.5"
```

Comment:
Do we need a dependency for this? Its entire code is under 300 lines, including tests.

Reply:
Strictly speaking, no... I could copy the code and include a license notice... I even considered just blindly dropping the prefix which would work 99.9% of the time.

But `dunce` seems to be a pretty safe dependency to add. It's fairly popular. It's used directly by rust-lang as a part of `miri`. And it is maintained by a developer that seems pretty active in the Rust community. In fact, [the same maintainer](https://crates.io/users/kornelski?sort=downloads) also maintains `security-framework`, `urlencoding`, and `xml-rs` (among many others) and these crates are already being used directly or indirectly by CLI.

Reply:
I personally would **not** have added a dependency for this. I don't consider it good practice for dependencies with essentially no code.

However since it's not some random tiny crate nobody else uses, I will leave the final judgement up to you. I just wanted to make sure it was carefully considered on your part.